encryptor/network/server_thread.py

```diff
@@ -15,11 +18,12 @@ class ServerThread(QThread):
     new_message = pyqtSignal(Message)
     disconnect = pyqtSignal()
 
-    def __init__(self, addr: Address) -> None:
+    def __init__(self, addr: Address, keys_dir: Path) -> None:
```

Comment:
Remove `keys_dir` from the server thread.

Reply:
done